src/puppetlabs/puppetdb/http/query.clj

```diff
@@ -304,48 +307,53 @@
   "Time how long it takes to transform a PQL query to AST, if the time
    is greater than 1s, a warning is logged. Returns the transformed
    query."
-  [query]
+  [query query-uuid log-queries?]
+  (when log-queries?
+    (log/info (trs "Parsing PDBQuery:{0}:{1}" query-uuid (pr-str query))))
   (let [start (System/nanoTime)
         result (pql/parse-pql-query query)
         elapsed (/ (- (System/nanoTime) start) 1000000.0)
         max-pql-limit 1000]
-    (when (> elapsed max-pql-limit)
-      (log/warn (trs "Parsing PQL took {0} ms: {1}" elapsed (pr-str query))))
+    (when (and log-queries? (> elapsed max-pql-limit))
+      (log/warn (trs "Parsing PQL took {0} ms for PDBQuery:{1}:{2}" elapsed query-uuid (pr-str query))))
```

Comment:
@austb One thing I'm not sure about: does it make sense to only print the PQL time logging when `log-queries` is true? Previously it always printed out but I feel like it should be behind that flag with the rest of the query logging